src/services/match.js

```diff
@@ -340,6 +340,27 @@ const removeExistingMatch = async(req, res, next) =>
     }
 }
 
+const cancelSendRequest = async (req, res, next) => 
+{
+    try 
+    {
+        const matchId = req.params.matchId;
+        const match = await Match.findById(matchId);
+
+        if (!match) return res.status(404);
```

Comment:
I understand why you chose 404 and while it maybe right, it won't seem right from the perspective of a front-end dev or some other developer using postman and encounter this 404. It would be difficult to figure out what this 404 means. Is it returned from the request/application handler that the endpoint does not exist? or the dudes request we are trying to remove is non existing one?

While this may not matter much for this specific project but this would be a good practice to specify what is this 404 about. Maybe returning a JSON with message saying that the match request does not exist.